geowave-vector/src/main/java/mil/nga/giat/geowave/vector/plugin/GeoWaveGTDataStore.java

```diff
@@ -112,6 +112,8 @@
 			LOGGER.error(
 					"Unable to decode EPSG:4326 CRS",
 					e);
+			throw new RuntimeException(
```

Comment:
pass along the factory exception?
ie. use constructor:
throw new RuntimeException(<String message>, e);